PYME/Acquire/ui/HDFSpoolFrame.py

```diff
@@ -259,11 +259,11 @@ def _spool_pan(self):
     
         hsizer.Add(self.bStopSpooling, 0, wx.ALL | wx.ALIGN_CENTER_VERTICAL, 5)
     
-        self.bAnalyse = wx.Button(self.spoolProgPan, -1, 'Analyse', style=wx.BU_EXACTFIT)
-        self.bAnalyse.Enable(False)
-        self.bAnalyse.Bind(wx.EVT_BUTTON, self.OnBAnalyse)
+        self.open_button = wx.Button(self.spoolProgPan, -1, 'Open', style=wx.BU_EXACTFIT)
```

Comment:
It probably makes sense to follow the control type prefix - e.g. `bBlah`, `stBlah` etc naming convention for wx gui controls (we should only really be striving for pep8 in stuff which could be considered part of an 'API' - i.e. called from other code).

Reply:
camelCase and short prefixes aren't always gfReadability. Why would we draw a line between making our API pep8 and the rest of our code (slowly, if we are already changing it, etc.)?